packages/experimental/Button/src/FabButton/FabButtonTokens.ts

```diff
@@ -0,0 +1,18 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
+import { TokenSettings } from '@fluentui-react-native/use-styling';
+import { ButtonCoreTokens } from '../Button.types';
+
+export const defaultFabButtonTokens: TokenSettings<ButtonCoreTokens, Theme> = () =>
+  ({
+    borderRadius: globalTokens.corner.radius.circle, // big number for always rounded corners
```

Comment:
Since this is a token now, we can probably remove the comment explainer

Reply:
Done with latest commits.